generator/Generator.scala

```diff
@@ -279,10 +279,25 @@ def generateMainClasses(): Unit = {
                   }
                 """)("\n\n")}
 
+                /**
+                 * <p>Provides a default value which is returned if no case matches.</p>
+                 * <p>Note that this method takes the default by value which means that the input is
+                 * <em>eagerly evaluated</em> even if the orElse clause of the expression is not executed.
+                 * Unless you already have a default value calculated or as a literal it might be better
+                 * to use the {@link Match.Case#orElse(Supplier)} alternative to gain lazy evaluation.</p>
+                 *
+                 * @param defaultValue The default value.
+                 * @return a Match-expression
+                 */
```

Comment:
I also fought with javadoc / doclint. and discovered that we can use for paragraphs also a single `<p>` as a kind of separator. This javadoc is totally ok!! I just wanted to mention it because it surprised me that a single `<p>` is allowed. Feels a little bit like invalid html. However, I use it in newer javadocs, because it is shorter.

These are the same:

```java
Some text.
<p>
More text.
```

```java
<p>Some text.</p>
<p>More text.</p>
```

Additionally code blocks and lists do not need no paragraph separator.

```java
Some text.

<pre>
<code>
// code
</code>
</pre>

More text.

<ul>
<li>item</li>
</ul>
```